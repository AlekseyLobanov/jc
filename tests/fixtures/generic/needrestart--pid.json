{"version":"3.6","pid":["at-spi-bus-laun=2031","at-spi2-registr=2073","dbus-daemon=1754,2037"]}
