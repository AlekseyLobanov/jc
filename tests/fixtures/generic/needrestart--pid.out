NEEDRESTART-VER: 3.6
NEEDRESTART-PID: at-spi-bus-laun=2031
NEEDRESTART-PID: at-spi2-registr=2073
NEEDRESTART-PID: dbus-daemon=1754,2037
